eole/train_single.py

```diff
@@ -188,6 +193,7 @@ def main(config, device_id):
     )
 
     if config.training.torch_compile:
+        torch._dynamo.config.cache_size_limit = 16
```

Comment:
Should this be configurable?

Reply:
I would leave it this way for now but maybe in the future.